user_management/src/user/views.py

```diff
@@ -0,0 +1,166 @@
+from django.http import JsonResponse
+from user.models import User
+from django.views import View
+from django.conf import settings
+from user_management.JWTManager import JWTManager
+import json
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+
+
+@method_decorator(csrf_exempt, name='dispatch')
+class SignUpView(View):
+    @csrf_exempt
+    def post(self, request):
+        try:
+            json_request = json.loads(request.body.decode('utf-8'))
+            validation_errors = self.signup_infos_validation(json_request)
+            if not validation_errors:
+                User.objects.create(username=json_request['username'],
```

Comment:
user = User.objects.create()
 User.objects.create() returns the created user instance. This avoids making an additional call to the DB